.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+<!--
+This form is for bug reports and feature requests ONLY!
+
+For high severity issues, please contact AWS Support directly.
+
+Please provide the following information:
+-->
+
+<!--
+BUG REPORT or FEATURE REQUEST ?
+
+Uncomment only one of the below items
+-->
+
+<!-- ### Bug Report -->
+<!-- ### Feature Request -->
+
+
+### Summary
+<!-- Please provide a brief outline of the bug or feature request -->
+
+
+### Description
+<!-- Provide detailed information about the issue -->
+
+
+<!-- -->
+### Expected Behavior
+
+
+### Supporting Log Snippets
+<!-- Please provide supporting log information for this issue -->
+
+
+### Stacktrace
+
+
+### Environment Details
```

Comment:
@richardpen The logs collector does most of this but we should still draw out the relevant sections here. 
This would also help customers who can't run the ecs-logs-collector.